File: CodeChef/Self Learning/Basic Programming Concepts/Programming_Languages.cpp
#include <bits/stdc++.h>
using namespace std;

void solve(){
    int A, B, A1, B1, A2, B2;

    cin >> A >> B >> A1 >> B1 >> A2 >> B2;
    set<int> one = {A1, B1};
    set<int> two = {A2, B2};

    int answer = 0;
    if(one.count(A) && one.count(B)){
        answer = 1;
    } else if(two.count(A) && two.count(B)){
        answer = 2;
    }
    cout << answer << '\n';
}

int main(){
    ios::sync_with_stdio(0);
    cin.tie(0);
    int t;
    cin >> t;
    while(t--){
        solve();
    }
}

File: CodeChef/Self Learning/Basic Programming Concepts/Programming_Languages.py
import sys
input = sys.stdin.readline

t = int(input())
for _ in range(t):
    A, B, A1, B1, A2, B2 = map(int, input().split())

    one = [A1, B1]
    two = [A2, B2]

    answer = 0
    if A in one and B in one:
        answer = 1
    elif A in two and B in two:
        answer = 2
    
    print(answer)